doc/OSAL-Configuration-Guide-Markdown.md

```diff
@@ -0,0 +1,798 @@
+[National Aeronautics and Space Administration]{.smallcaps}
+
+![][1]
+
+**Operating System Abstraction Layer (OSAL)**
+
+Configuration Guide
+
+582-2007-00
+
+January 31, 2016 (Version 4.2)
+
+**[Authors:]{.ul}**
+
+![][2]
+
+![][3]
+
+**[Approvals:]{.ul}**
+
+![][4]
+
+Acknowledgements
+
+Revision History
+
+| Revision Number |  Release Date | Changes to Prior Revision  | Approval |
```

Comment:
Maybe move these to an appendix of sorts? I'm not sure we need this when the document is versioned using git